app/src/main/java/com/asfoundation/wallet/repository/NoValidateTransactionValidatorOnChain.java

```diff
@@ -0,0 +1,20 @@
+package com.asfoundation.wallet.repository;
+
+import com.asfoundation.wallet.interact.SendTransactionInteract;
+import io.reactivex.Completable;
+
+public class NoValidateTransactionValidatorOnChain implements TransactionValidator {
+
+  private final SendTransactionInteract sendTransactionInteract;
+  private final TrackTransactionService pendingTransactionService;
+
+  public NoValidateTransactionValidatorOnChain(SendTransactionInteract sendTransactionInteract,
```

Comment:
OnChain is not relevant here, consider rename this class to NoValidateTransactionValidator